Commands/Modules/Converter.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Discord;
+using Discord.Interactions;
+using FFMpegCore;
+using Serilog;
+using SixLabors.ImageSharp;
```

Comment:
Gonna have to add the licence (apache 2.0)

Just putting it here so I don't forget (hopefully lol)